File: rules/javascript/tests/test_mass_assignment_tn.js
// True negative: explicit field pick (safe)
const express = require('express');

app.post('/users', async (req, res) => {
    // ok: ez-express-mass-assignment
    const user = await User.create({
        name: req.body.name,
        email: req.body.email,
    });
    res.json(user);
});

File: rules/python/tests/test_mass_assignment_tn.py
# True negative: explicit field assignment (safe)
from django.http import JsonResponse

def create_user(request):
    # ok: ez-django-mass-assignment
    User.objects.create(
        username=request.POST.get("username"),
        email=request.POST.get("email"),
    )
    return JsonResponse({"ok": True})
